### ISSUE TITLE
Upgrade from Cordova 4.x

### ISSUE THREAD

**Issue by rgamez** [edina/fieldtrip-cobweb-project#136]
After upgrading to Xcode 7 I tried to build the project and it failed because some API calls are not longer supported, the solution was to upgrade Cordova 5, but for the same reason I was forced to upgrade the cordova plugins, which breaks some of fieldtrip plugins:

1. Offline maps and Overlays
  Our fork of the [sqlite cordova](https://github.com/edina/Cordova-SQLitePlugin.git) plugin is well behind the official repository, I tested releases compatible with Cordova 5 from the master and legacy branch but the BLOB type is not longer supported [[1]](https://github.com/litehelpers/Cordova-sqlite-legacy/blob/legacy-master/CHANGES.md#0711), [[2]](https://github.com/litehelpers/Cordova-sqlite-storage/blob/storage-master/CHANGES.md#cordova-sqlite-storage-122), [[3]](https://github.com/litehelpers/Cordova-sqlite-storage/blob/storage-master/CHANGES.md#cordova-sqlite-storage-122), also some API changes had taken place.
Alternatives may require significant effort to preserve the compatibility with Android and iOS, I believe @murrayk and me ([http-mbtiles](https://github.com/rgamez/cordova-plugin-http-mbtiles)) made some experiments serving the tiles via an embedded http server but is a work in progress and not a multiplatform cordova plugin.

2. Map search and Maps
  The need to update our http call to https It is not an specific problem of the upgrade but it is more evident as insecure endpoints are blocked by default in iOS, the https support is already available in some of the services we use (cobweb and fieldtripgb):

| Service               | https         |
| ---------------------- |:-------------:|
| openstreetmaps  |        x       |
| openstream         |                 |
| opencyclemap     |        x       | 
| unlock                  |                 | 
| geonames*           |        x      | 
*self-signed certificate
3. Icons are being renamed for iOS also new sizes and resolutions are being added to both platforms.

In conclusion:

Until the sqlite issue is resolved we need to continue supporting Cordova 4 and in the case of iOS use Xcode 6 for building the app (as is reflected now in the [documentation](https://github.com/edina/fieldtrip-open/wiki/install#ios))
